ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/datasource/metadata/tree/TreeNodeDirectory.java

```diff
@@ -0,0 +1,9 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.metadata.tree;
+
+public interface TreeNodeDirectory extends TreeNode {
+
+    public void addLeaf(String name);
+
+    public TreeNodeDirectory getDirectory(String name);
```

Comment:
Access modifier is 'public' is redudant here.  In interfaces access modifier by default is public

Reply:
changed